App/Sagas/NodeStarted.ts

```diff
@@ -11,7 +11,6 @@ export function * onNodeStarted () {
     try {
       yield all([
         put(ThreadsActions.refreshThreadsRequest()),
-        call(refreshTokens),
```

Comment:
If you get rid of this, how does the app get the tokens before the first upload is attempted? We're not persisting tokens. Should we?

Reply:
in the upload function, it will check if they are undefined and grab them before attempting

Reply:
that works